app/jobs/in_person/send_proofing_notification_job.rb

```diff
@@ -0,0 +1,107 @@
+# frozen_string_literal: true
+
+module InPerson
+  class SendProofingNotificationJob < ApplicationJob
+    # @param [InPersonEnrollment] enrollment
+    def perform(enrollment_id)
```

Comment:
is the param the enrollment object or just its id? the documentation and param name disagree

Reply:
@zachmargolis just id, fixed it.